HELP.md

```diff
@@ -0,0 +1,124 @@
+# Overview
+
+- [Installation](#installation)
+  - [cnquery](#cnquery)
+  - [cnspec](#cnspec)
+- [Package Information](#package-information)
+- [Kubernetes](#kubernetes)
+
+# Installation
+
+## cnquery
+
+###  via Shell Script (Linux and macOS)
+
+[`https://install.mondoo.com/sh/cnquery`](https://install.mondoo.com/sh/cnquery)
+
+```bash
+bash -c "$(curl -sSL https://install.mondoo.com/sh/cnquery)"
+```
+
+To login with Mondoo Platform:
+
+```bash
+cnquery login -t 'eyJh...llZ4BW'
+```
+
+### via Powershell (Windows)
+
+[`https://install.mondoo.com/ps1/cnquery`](https://install.mondoo.com/ps1/cnquery)
+
+```powershell
+Set-ExecutionPolicy Unrestricted -Scope Process -Force;
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
+iex ((New-Object System.Net.WebClient).DownloadString('https://install.mondoo.com/ps1/cnquery'));
+Install-Mondoo -Product cnquery;
+```
+
+## cnspec
+
+###  via Shell Script (Linux and macOS)
+
+[`https://install.mondoo.com/sh/cnspec`](https://install.mondoo.com/sh/cnspec)
+
+```bash
+bash -c "$(curl -sSL https://install.mondoo.com/sh/cnspec)"
+```
+
+To login with Mondoo Platform:
+
+```bash
+cnspec login -t 'eyJh...llZ4BW'
+```
+
+### via Powershell (Windows)
+
+[`https://install.mondoo.com/ps1/cnspec`](https://install.mondoo.com/ps1/cnspec)
+
+```powershell
+Set-ExecutionPolicy Unrestricted -Scope Process -Force;
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
+iex ((New-Object System.Net.WebClient).DownloadString('https://install.mondoo.com/ps1/cnspec'));
+Install-Mondoo -Product cnspec;
+```
+
+# Package Information
+
+`https://install.mondoo.com/package/mondoo/{platform}/{arch}/{filetype}/{version}/{method}`
+
+The arguments support the following values
+  - **platform**: linux, windows, darwin
+  - **arch**:     amd64, arm64, armv7, armv6, 386
```

Comment:
We also have `ppc64le`. Is this also supported in the installer?

Reply:
100% great catch